### PR TITLE
-added basic single command symfony/console implementation

### PR DESCRIPTION
**Pull request by JEM94**

### CODE REVIEW

File: bin/generate-tests.php [hunk 1]
```diff
@@ -11,4 +15,16 @@
     )
 );
 
-Controller::init()->run();
+(new Application('echo', '1.0.0'))
```
Comment (JEM94) [inline bin/generate-tests.php]: schon gesehen :D

File: composer.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
         "twig/twig": "^1.35.0",
         "nikic/php-parser": "^1.4.1",
         "symfony/finder": "^2.6.13",
+        "symfony/console":"^4.0.0",
```
Comment (Idrinth) [inline composer.json]: 4.0.0 is not going to work on php 5.3

File: bin/generate-tests.php [hunk 3]
```diff
@@ -11,4 +15,16 @@
     )
 );
 
-Controller::init()->run();
+(new Application('PHPUnit-Test-Generator', '1.0.0'))
+    ->register('generate-tests')
```
Comment (Idrinth) [inline bin/generate-tests.php:19]: please register a command class, so that this file here has as little logic as possible.

**Review by Idrinth — CHANGES_REQUESTED** [review]

**Comment by Idrinth** [comment]
Just to give a reason for why I'd like the command to be a class(extending symfony's command class):
- it's hard to test procedural code automatically
- Once #9 is going to happen, we'd need to put even more untestable logic in the procedural code
- it is very likely, that the complexity of the command will increase, making tests more important

On a technical basis the implementation looks ok, the main issue is how hard it's going to be to test it.

File: bin/generate-tests.php (hunk 3 above)
Comment (Idrinth) [reply bin/generate-tests.php:19]: and add defaults

**Comment by Idrinth** [comment]
@JEM94 should we close this and delete the branch? we're waiting on idrinth/simple-console#2 after all

**Pull request closed without merging**